__tests__/app.test.js

```diff
@@ -95,4 +95,64 @@ describe('/api/articles', () => {
       })
     
   })
+describe('POST: api/articles/:article_id/comments', () => {
+    test('POST: 201, inserts new comment for an article into the database and sends back the added comment to the client', () => {
+        const newComment = {
+            username: 'rogersop',
+            body: 'Amazing article!'
+        }
+        return request(app)
+            .post('/api/articles/1/comments')
+            .send(newComment)
+            .expect(201)
+            .then(({body}) => {
+                console.log(body)
+                expect(body.comment).toHaveProperty('comment_id', expect.any(Number))
+                expect(body.comment.body).toBe('Amazing article!') 
+                expect(body.comment.article_id).toBe(1)
+                expect(body.comment.author).toBe('rogersop')
+                expect(body.comment).toHaveProperty('votes', expect.any(Number))
+                expect(body.comment).toHaveProperty('created_at', expect.any(String))
+            })
+    })
+    test('POST:404 responds with an appropriate error message when provided with an valid but non existent article number', () => {
```

Comment:
What happens if the username is not a existing username? It'd be good to see a test for this.